castle/cms/tests/robot/test_files.robot

```diff
@@ -0,0 +1,79 @@
+*** Settings ***
+
+ Force Tags  wip-not_in_docs
+
+ Resource   plone/app/robotframework/selenium.robot
+ Resource   keywords.robot
+ Resource   common.robot
+
+ Test Setup  Open test browser
+ Test Teardown  Close all browsers
+
+*** Test Cases ***
+
+Test add files
+   Given a logged-in site administrator
+   Go to  ${PLONE_URL}
+
+   Set Selenium Speed  .1
+
+   Wait Until Page Contains Element  css=.introjs-skipall
```

Comment:
Does this work for you reliably? I've had a lot of trouble dealing with that damn popup

Reply:
sigh yes, I'm trying to fix this right now :(